packages/compass-components/src/components/generative-ai/generative-ai-input.tsx

```diff
@@ -150,6 +151,8 @@ type GenerativeAIInputProps = {
   isFetching?: boolean;
   placeholder?: string;
   show: boolean;
+  isAggregationGeneratedFromQuery?: boolean;
```

Comment:
A nit, fine as is, if we have the value in the store not as possibly undefined I prefer to keep the component interfaces the same.
```suggestion
  isAggregationGeneratedFromQuery: boolean;
```
That way if something changes in the store we're for sure to catch it here. In tests we can then default it to `false`.